### PR TITLE
fix(MyOrganization & social): fixed the edit logo in MyOrg and added indexing in techtag

### PR DESCRIPTION
**Pull request by abhishek-mishra019**
Signed-off-by: Abhishek Mishra [am1426620@gmail.com](mailto:am1426620@gmail.com)
### Functionality:
to improve functionalities in My organization page

### Solution:
fixed the edit org logo functionality, and added firebase indexing for techtag, also removed unnecessary console logs.
also fixed start service errors

### Risk level:
- [x] high 
- [ ] medium
- [] low

### How to test:
try the my organization page in dark mode and also try to upload a logo and check in firestore

### CODE REVIEW

File: functions/model/librarian/tark/getFilesinOrganization.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ * modify it under the terms of the MIT License
+ *
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the MIT License for more details.
+ ***********************************************************/
+
+const e = require("cors");
```
Comment (viv9k) [inline functions/model/librarian/tark/getFilesinOrganization.js:19]: remove this

File: src/app/body/social-page/posts/posts.component.css [hunk 2]
```diff
@@ -1,3 +1,7 @@
+.post {
+  font-family: 'Plus Jakarta Sans', sans-serif;
```
Comment (viv9k) [inline src/app/body/social-page/posts/posts.component.css:2]: --font can be used here

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the required changes

File: functions/model/librarian/tark/getFilesinOrganization.js (hunk 1 above)
Comment (abhishek-mishra019) [reply functions/model/librarian/tark/getFilesinOrganization.js]: resolved

File: src/app/body/social-page/posts/posts.component.css (hunk 2 above)
Comment (abhishek-mishra019) [reply src/app/body/social-page/posts/posts.component.css]: resolved

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**